build.gradle.kts

```diff
@@ -298,7 +298,15 @@ license {
 ktlint {
     enableExperimentalRules.set(true)
 }
-
+// fun String.execute(currentWorkingDir: File = file("./")): InputStreamReader {
```

Comment:
This commented out code isn't needed anymore, right?

Reply:
Nah

Reply:
nope